shift

```diff
@@ -167,15 +168,7 @@ sub_attach() { # <service>        - Run bash on a running service
 }
 
 sub_compose() { # <cmd...>        - Run a compose with associated files
-    OS=`uname`
-    if [ "$OS" == "Darwin" ] ; then
-	    docker compose $@
-    elif [ "$OS" == "Linux" ] ; then 
-	    docker-compose $@
-    else
-	    echo "not sure how to handle your OS' version of docker.  One of the above should work!"
-	    exit 255
-    fi
+    docker-compose $@
```

Comment:
We originally did this `docker compose` vs `docker-compose` dance to work around some other failures, but that was in 2021/2022 and Docker v20.10 — see commit notes here: 
https://github.com/shift-org/shift-docs/commit/32d9bd225b357d6b45701c3ea9342c02a1bb2cb4
https://github.com/shift-org/shift-docs/commit/7ab6a19e1a53a6015d27cb6a435f938b2710265b

As long as this runs in Docker v24 or later on Ubuntu (production) and macOS (both Apple Silicon and Intel), we're all good.

Reply:
Theoretically it might also run on Windows, but we haven't actively developed/tested that. If it fails on Windows it could be for other reasons and wouldn't be a blocker.

Reply:
re: production, because it's 'linux' it currently uses the "docker-compose"; so it won't see any change.
re: windows, because this a bash script, it will only run under "linux on windows" anyway, which it sees as 'linux' so it won't see any change. ( someday: i'd love to convert this to a npm script, so that linux on windows isn't needed. )

all that said, it's definitely bad practice to do two things at once. i was just being lazy. i'll make a separate pr for this, and pull ( oh, no puns ) this out.